VERSION.cmake

```diff
@@ -1,4 +1,4 @@
-set (HIF_MAJOR_VERSION 0)
-set (HIF_MINOR_VERSION 7)
-set (HIF_MICRO_VERSION 0)
-set (HIF_VERSION ${HIF_MAJOR_VERSION}.${HIF_MINOR_VERSION}.${HIF_MICRO_VERSION})
+set (DNF_MAJOR_VERSION 0)
+set (DNF_MINOR_VERSION 7)
+set (DNF_MICRO_VERSION 0)
+set (DNF_VERSION ${DNF_MAJOR_VERSION}.${DNF_MINOR_VERSION}.${DNF_MICRO_VERSION})
```

Comment:
Can we use `LIBDNF` instead of `DNF` here? That way there's no chance of a namespace collision.